src/python/national_voter_file/us_states/vt/transformer.py

```diff
@@ -0,0 +1,268 @@
+            'GRAND ISLE': 'GI',
+            'LAMOILLE': 'LAM',
+            'ORANGE': 'ORA',
+            'ORLEANS': 'ORL',
+            'RUTLAND': 'RUT',
+            'WASHINGTON': 'WAV',
+            'WINDHAM': 'WDH',
+            'WINDSOR': 'WDR',
+        }
+        return {'COUNTYCODE': county_code_map[input_dict.get('County')]}
+
+    def extract_mailing_address(self, input_dict):
+        """
+        Relies on the usaddress package.
+
+        Inputs:
+            input_dict: dictionary of form {colname: value} from raw data
+        Outputs:
+            Dictionary with following keys
+                'MAIL_ADDRESS_LINE1'
+                'MAIL_ADDRESS_LINE2'
+                'MAIL_CITY'
+                'MAIL_STATE'
+                'MAIL_ZIP_CODE'
+                'MAIL_COUNTRY'
+        """
+        columns = [
+            'Mailing Address Line 1',
+            'Mailing Address Line 2',
+            'Mailing Address City',
+            'Mailing Address State',
+            'Mailing Address Zip'
+        ]
+
+        mail_str = ' '.join([input_dict[x] for x in columns if input_dict[x] is not None])
+        usaddress_dict, usaddress_type = self.usaddress_tag(mail_str)
+
+        mail_addr_dict = {}
+
+        if usaddress_type in ['Ambiguous', None]:
+            print('Warn - {}: Ambiguous mailing address, falling back to residential'.format(usaddress_type))
+        else:
+            mail_addr_dict = {
+                'MAIL_ADDRESS_LINE1': self.construct_mail_address_1(
+                    usaddress_dict,
+                    usaddress_type,
+                ),
+                'MAIL_ADDRESS_LINE2': self.construct_mail_address_2(usaddress_dict),
+                'MAIL_CITY': usaddress_dict.get('PlaceName', None),
+                'MAIL_ZIP_CODE': usaddress_dict.get('ZipCode', None),
+                'MAIL_STATE': usaddress_dict.get('StateName', None),
+                'MAIL_COUNTRY': 'USA',
+            }
+
+        return mail_addr_dict
+
+
+    #### Political methods #####################################################
+
+    def extract_state_voter_ref(self, input_dict):
+        return {'STATE_VOTER_REF' : 'VT' + input_dict['VoterID']}
+
+    def extract_registration_date(self, input_columns):
+        """
+        Inputs:
+            input_columns: name or list of columns
+        Outputs:
+            Dictionary with following keys
+                'REGISTRATION_DATE'
+        """
+        return {'REGISTRATION_DATE': self.convert_date(input_columns['Date of Registration'])}
+
+    def extract_congressional_dist(self, input_columns):
+        """
+        Inputs:
+            input_columns: name or list of columns
+        Outputs:
+            Dictionary with following keys
+                'CONGRESSIONAL_DIST'
+        """
+
+        # Vermont only has 1 at-large Congressional District. This is the CensusReporter 
+        # GeoID for the VT At-large CD.
+        return {'CONGRESSIONAL_DIST': '50000US5000'}
```

Comment:
Is this the default value you guys were looking for -- you can see this is the Census Reporter GeoID for VT At-large here: https://api.censusreporter.org/1.0/geo/tiger2015/50000US5000